pkg/env/costmodelenv.go

```diff
@@ -35,6 +35,7 @@ const (
 	CSVEndpointEnvVar              = "CSV_ENDPOINT"
 	CSVPathEnvVar                  = "CSV_PATH"
 	ConfigPathEnvVar               = "CONFIG_PATH"
+	LabelConfigPathEnvVar 		   = "LABELCONFIG_PATH"
```

Comment:
Nit: A linter would line these up for you-- can you make this const declaration equal sign align with the rest?

Reply:
Weird, I am not seeing this back locally. Did you already change it?